### ISSUE TITLE
TRE Documentation: All Of Us Researcher Workbench, UK Biobank Research Analysis Platform

### ISSUE THREAD

**Issue by smlmbrt** [PGScatalog/pgsc_calc#391]
We're currently developing documentation with clear intsructions for how to run the pgsc_calc tool in specific trusted research environments ([TRE](https://www.hdruk.ac.uk/access-to-health-data/trusted-research-environments/)). This includes:

- [ ] All Of Us (AoU) [Researcher Workbench](https://workbench.researchallofus.org/login)
- [ ] UK Biobank (UKB) [Research Analysis Platform](https://www.ukbiobank.ac.uk/enable-your-research/research-analysis-platform)

We'll keep this ticket open for people who have insights or questions about the process.

**Comment by nebfield** [comment]
[Draft instructions that describe using a single container image](https://docs.google.com/document/d/1zis-OVDoOrk_LSe3VOxi_Qmy1ByCPGo_to8U8PRv7k8/edit?usp=sharing)

**Comment by Fiwx** [comment]
Is it possible to run pgsc_calc, containerized, on All of Us?

**Comment by smlmbrt** [comment]
@Fiwx, yes it is possible to run it inside a docker image and a single job. We are currently finalising some documentation to describe this process, and planning some future software updates that should make the pipeline easier to run in theses settings.

**Comment by Fiwx** [comment]
That's great! If you have any code or notebooks that are able to be shared on that, I would appreciate it, even if they're unfinished or not perfect. I've been trying to do that for a long time and haven't been able to figure it out.

**Comment by nebfield** [comment]
closed by #402 

[docs published here](https://pgsc-calc.readthedocs.io/en/latest/how-to/AOU.html)
